Add description of spatial files to download files section (#66)
AlexsLemonade/scpca-docs: docs/download_files.md

Every download also includes a single `single_cell_metadata.tsv` file containing metadata for all libraries included in the download.

> If a sample includes a library that was processed using spatial transcriptomics, an additional folder, `SCPCL000000_spatial`, will be included inside the sample folder. 
> This folder will contain all output files related to processing the spatial transcriptomics library. 
> See the [description of the Spatial transcriptomics output section below](#download-contents-for-samples-with-spatial-transcriptomics) for more information. 

Reviewer: I like having this section here, but I wanted to try to make it a bit shorter. I also updated the link to match the change I made to the header (but you may need to update it again if anything changes there!). I still don't love it, but it may just be an awkward concept to get across succinctly.

```suggestion
If a sample includes a library that was processed using spatial transcriptomics, there will be an additional folder in the sample directory, `SCPCL000000_spatial`. 
For more information about the contents of this folder, see the [spatial transcriptomics libraries section below](#spatial-transcriptomics-libraries). 
```

See the {ref}`Experiment metadata <sce_file_contents:experiment metadata>` section for more information on metadata columns that can be found in this file as well as inside the `SingleCellExperiment` object.

For projects with bulk RNA-seq data, the `bulk_metadata.tsv` file will be included for project downloads. 
This file will contain fields equivalent to those found in the `single_cell_metadata.tsv` related to processing the sample, but will not contain patient or disease specific metadata (e.g. `age`, `sex`, `diagnosis`, `subdiagnosis`, `tissue_location`, or `disease_timing`).

## Download contents for samples with spatial transcriptomics

For all spatial transcriptomics libraries, a `SCPCL000000_spatial` folder will be nested inside the corresponding sample folder in the download. 

> Inside that folder will be the following folders and files: 
> - A folder containing the [unfiltered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices): `raw_feature_bc_matrix`,
> - A folder containing the [filtered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices): `filtered_feature_bc_matrix`,

Reviewer: I reorganized to move the folder names to the start of the lines.
I am not sure the extra line is required, but it is good form for markdown to avoid potential ambiguity.
```suggestion
Inside that folder will be the following folders and files: 

- A `raw_feature_bc_matrix` folder containing the [unfiltered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices)
- A `filtered_feature_bc_matrix` folder containing the [filtered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices)
```

> - A folder containing the [spatial files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/images): `spatial`,

Reviewer: I don't like long links, but I think a bit of context for what the `spatial` folder contains would be useful here. I also moved the folder name to the front of the bullet point (a suggestion I am going to make for each one)

```suggestion
- A `spatial` folder containing [images and position information](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/images)
```

> - A [summary html report](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/summary), as output by Space Ranger: `SCPCL000000_spaceranger_summary.html`

Reviewer: Reorganizing for parallel construction as with other edits
```suggestion
- A `SCPCL000000_spaceranger_summary.html` file containing the Space Ranger [summary html report](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/summary)
```

> A full description of each of file included in the download for spatial transcriptomics libraries can also be found in the [documentation for the `spaceranger count` function](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/using/count#outputs). 

Reviewer: Just shortening the sentence/link text.
```suggestion
A full description of each of files included in the download for spatial transcriptomics libraries can also be found in the [`spaceranger count` documentation](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/using/count#outputs). 
```
